### ISSUE TITLE
Problem with dlmalloc v0.2.5

### ISSUE THREAD

**Issue by obioma** [Synphonyte/leptos-use#82]
Hi,
after updating the complete rust tool chain the following error happened when a message was received. I'm not sure if the real root cause lies within leptos_use or rather somewhere else.

What I can say is that after manually enforcing dlmalloc v0.2.4 via an entry in cargo.toml and using

```rust 
//main.rs
#[global_allocator]
static ALLOCATOR: dlmalloc::GlobalDlmalloc = unsafe { dlmalloc::GlobalDlmalloc};
```
it did work again. Utilising 
```rust 
#[global_allocator]
static ALLOCATOR: talc::TalckWasm = unsafe { talc::TalckWasm::new_global() };
```
resulted in an similar error - also with a validation assertion. 

While it is working now, I think it is worth to understand the underlying root cause, but I must admit I'm not able to dig deeper to to lack of knowledge. Maybe somebody from leptos_use or even leptos community has an idea.

Thx

Here the error message.

```
panicked at /rust/deps/dlmalloc-0.2.5/src/dlmalloc.rs:1182:13:
assertion failed: psize <= size + max_overhead

Stack:

__wbg_get_imports/imports.wbg.__wbg_new_abda76e883ba8a5f@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:514:17
q_wasm-832dc33b11cb2c78.wasm.console_error_panic_hook::Error::new::hb2f3e4cfe3567c8e@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[17877]:0xbf774d
q_wasm-832dc33b11cb2c78.wasm.console_error_panic_hook::hook_impl::had23a6460ed76737@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[4790]:0x9ac935
q_wasm-832dc33b11cb2c78.wasm.console_error_panic_hook::hook::hbffec8d079d0a1a5@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[21745]:0xc32ce5
q_wasm-832dc33b11cb2c78.wasm.core::ops::function::Fn::call::h0737598532f8800f@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[19150]:0xc0d835
q_wasm-832dc33b11cb2c78.wasm.std::panicking::rust_panic_with_hook::he3f5031942851a57@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[8889]:0xad5206
q_wasm-832dc33b11cb2c78.wasm.std::panicking::begin_panic_handler::{{closure}}::h2fbd6f5ef7cde01d@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[11254]:0xb45855
q_wasm-832dc33b11cb2c78.wasm.std::sys_common::backtrace::__rust_end_short_backtrace::h2a11135b2d55c7a5@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[25171]:0xc53621
q_wasm-832dc33b11cb2c78.wasm.rust_begin_unwind@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[20221]:0xc1e342
q_wasm-832dc33b11cb2c78.wasm.core::panicking::panic_fmt::hd2edc01644c13cb8@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[21307]:0xc2d2a8
q_wasm-832dc33b11cb2c78.wasm.core::panicking::panic::h1654f8fcddbbeba3@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[19109]:0xc0cdcf
q_wasm-832dc33b11cb2c78.wasm.__rdl_dealloc@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[15166]:0xbc0505
q_wasm-832dc33b11cb2c78.wasm.__rust_dealloc@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[24956]:0xc52c36
q_wasm-832dc33b11cb2c78.wasm.<alloc::alloc::Global as core::alloc::Allocator>::deallocate::h0c91dd792980f986@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[9046]:0xaddaa9
q_wasm-832dc33b11cb2c78.wasm.<alloc::raw_vec::RawVec<T,A> as core::ops::drop::Drop>::drop::h5b9d7d345726ee19@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[9584]:0xaf92bc
q_wasm-832dc33b11cb2c78.wasm.core::ptr::drop_in_place<alloc::raw_vec::RawVec<u8>>::he35180a37260bb69@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[22509]:0xc3ca59
q_wasm-832dc33b11cb2c78.wasm.core::ptr::drop_in_place<alloc::vec::Vec<u8>>::hddfad45426af894f@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[20666]:0xc2483f
q_wasm-832dc33b11cb2c78.wasm.core::ptr::drop_in_place<alloc::string::String>::h1a4593956965df05@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[22503]:0xc3c91c
q_wasm-832dc33b11cb2c78.wasm.core::ptr::drop_in_place<core::option::Option<alloc::string::String>>::h67383d3331c28c7a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[13494]:0xb93c8d
q_wasm-832dc33b11cb2c78.wasm.<leptos_reactive::signal::WriteSignal<T> as leptos_reactive::signal::SignalSet>::set::{{closure}}::h2a36df7f3dcdfa85@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[11998]:0xb62f4d
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::signal::<impl leptos_reactive::node::NodeId>::update::{{closure}}::h7c9d30eccb53248c@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[1553]:0x7416ef
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::with_runtime::{{closure}}::hdb6c736cf0885cfd@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[11004]:0xb3b095
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::try_with::h657f2396a3a9f258@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[5370]:0x9e2ada
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::with::h95a9fba8de0c1105@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[7621]:0xa8a25f
q_wasm-832dc33b11cb2c78.wasm.<leptos_reactive::signal::WriteSignal<T> as leptos_reactive::signal::SignalSet>::set::h12931e7774ba169a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[426]:0x295bc4
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::{{closure}}::{{closure}}::{{closure}}::{{closure}}::h96346e4993639d2a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[3462]:0x915ceb
q_wasm-832dc33b11cb2c78.wasm.core::result::Result<T,E>::map_or_else::h8bb21178d0a3cb36@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[10424]:0xb214fd
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::{{closure}}::{{closure}}::{{closure}}::h0560601db6cfd88e@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[10345]:0xb1db4e
q_wasm-832dc33b11cb2c78.wasm.core::result::Result<T,E>::map_or_else::h0c9cd0fb5507ce99@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[7761]:0xa930e0
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::{{closure}}::{{closure}}::hb56977da34656958@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[5170]:0x9d08db
q_wasm-832dc33b11cb2c78.wasm.<dyn core::ops::function::FnMut<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::hbecea431dc2e5d7c@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[8871]:0xad41eb
__wbg_adapter_66@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:278:10
real@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:202:20
EventHandlerNonNull*__wbg_get_imports/imports.wbg.__wbg_setonmessage_3df13fd356f531d6@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:1126:33
q_wasm-832dc33b11cb2c78.wasm.web_sys::features::gen_WebSocket::WebSocket::set_onmessage::he7b2f30eafa37f41@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[14034]:0xba34f2
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::{{closure}}::hcd6b3c6a3133f59b@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[1084]:0x5e2350
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::{{closure}}::h40f3a60d0964c00c@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[3493]:0x919e68
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::{{closure}}::h61d9059d301b169f@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[15348]:0xbc4768
q_wasm-832dc33b11cb2c78.wasm.<leptos_reactive::effect::EffectState<T,F> as leptos_reactive::effect::AnyComputation>::run::hea90b26eff62c89a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[731]:0x480176
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::Runtime::update::{{closure}}::h9be4d756a121d114@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[8347]:0xab6a96
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::Runtime::with_observer::h273422681ecfd4c7@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[5729]:0xa01cc9
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::Runtime::update::h9a2d982b3f2e1cb1@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[1697]:0x78691a
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::Runtime::update_if_necessary::hcddb21245ba372dc@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[2216]:0x833590
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::effect::create_effect::{{closure}}::{{closure}}::{{closure}}::h960fb90f79bdb691@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[16458]:0xbdc1e0
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::with_runtime::{{closure}}::h137e39a4c1692857@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[17892]:0xbf7bdb
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::try_with::h90efb462ea0abac2@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[8115]:0xaa8d70
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::with::h8e761d36fbe04509@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[10617]:0xb2a22b
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::effect::create_effect::{{closure}}::{{closure}}::h96c7c1f01dc9410c@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[1445]:0x70172d
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::try_with_owner::{{closure}}::h6f0b54a695f1f8f6@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[2265]:0x83f23f
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::with_runtime::{{closure}}::ha8cae4f401ca89a3@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[12005]:0xb63372
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::try_with::h39e6fc0c5be37837@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[5428]:0x9e7d37
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::with::h216ef5abe3241627@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[7189]:0xa6daf6
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::try_with_owner::hc4f0583e08f47d76@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[1017]:0x5a6fe8
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::with_owner::hf341fcd1dbca324a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[7313]:0xa75f07
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::effect::create_effect::{{closure}}::hd691741a2dfaec90@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[11205]:0xb43715
q_wasm-832dc33b11cb2c78.wasm.<T as wasm_bindgen::closure::WasmClosureFnOnce<(),R>>::into_js_function::{{closure}}::h8da7652f5bff6173@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[2872]:0x8bb1cd
q_wasm-832dc33b11cb2c78.wasm.<dyn core::ops::function::FnMut<()>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::hb53e840e3f84e36a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[10622]:0xb2a633
__wbg_adapter_45@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:229:10
real@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:202:20
VoidFunction*__wbg_get_imports/imports.wbg.__wbg_call_67f2111acd2dfdb6/<@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:1297:33
handleError@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:318:18
__wbg_get_imports/imports.wbg.__wbg_call_67f2111acd2dfdb6@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6.js:1296:63
q_wasm-832dc33b11cb2c78.wasm.js_sys::Function::call1::hc33b9397988fdd9b@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[5347]:0x9e08e7
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::spawn_microtask::queue_microtask::h33d3c1db7f31228c@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[3790]:0x94070e
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket_with_options::hbabfde804a9e1f3d@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[332]:0x1cf828
q_wasm-832dc33b11cb2c78.wasm.leptos_use::use_websocket::use_websocket::h9765196c8376300a@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[17100]:0xbe8dce
q_wasm-832dc33b11cb2c78.wasm.q_wasm::pages::game::__GamePlay::h45a0faa61ec59867@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[230]:0x8003e
q_wasm-832dc33b11cb2c78.wasm.q_wasm::pages::game::GamePlay::{{closure}}::hc20197fa7bb93b13@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[6830]:0xa551bd
q_wasm-832dc33b11cb2c78.wasm.<leptos_dom::components::Component<F,V> as leptos_dom::IntoView>::into_view::{{closure}}::hc67c8c5f8923c4a2@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[6213]:0xa28369
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::RuntimeId::untrack::{{closure}}::h3e09b7aec420e9e1@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[3802]:0x941ee2
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::with_runtime::{{closure}}::hcd118e7dbcf0498b@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[8714]:0xacb689
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::try_with::hbb548fe0e96da05c@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[4022]:0x95c12d
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::with::hedc8f53111c18274@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[8217]:0xaaef42
q_wasm-832dc33b11cb2c78.wasm.<leptos_dom::components::Component<F,V> as leptos_dom::IntoView>::into_view::haae11731b73865bb@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[275]:0x13dbd3
q_wasm-832dc33b11cb2c78.wasm.<Func as leptos::ComponentConstructor<P>>::construct::h670296f7832d8995@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[5974]:0xa15d72
q_wasm-832dc33b11cb2c78.wasm.leptos::component_view::he7717ef2cd70e627@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[20341]:0xc1feb2
q_wasm-832dc33b11cb2c78.wasm.q_wasm::pages::game::__Game::h60b816351de4b7f9@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[672]:0x439b30
q_wasm-832dc33b11cb2c78.wasm.q_wasm::pages::game::Game::{{closure}}::h566667ceb2dfd420@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[23167]:0xc44acb
q_wasm-832dc33b11cb2c78.wasm.<leptos_dom::components::Component<F,V> as leptos_dom::IntoView>::into_view::{{closure}}::h8ddbdf0d801bda87@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[20013]:0xc1b0ac
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::RuntimeId::untrack::{{closure}}::h3807bac0659a1036@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[6848]:0xa565df
q_wasm-832dc33b11cb2c78.wasm.leptos_reactive::runtime::with_runtime::{{closure}}::hd6182a5b4155ea72@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[17391]:0xbee686
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::try_with::h5cd33e8b454bfe8b@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[7691]:0xa8ea46
q_wasm-832dc33b11cb2c78.wasm.std::thread::local::LocalKey<T>::with::hc2040e824c71a274@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[7949]:0xa9eb6d
q_wasm-832dc33b11cb2c78.wasm.<leptos_dom::components::Component<F,V> as leptos_dom::IntoView>::into_view::h6a37f4d41808ffbf@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[307]:0x1915cd
q_wasm-832dc33b11cb2c78.wasm.leptos_router::components::route::__Route::{{closure}}::hb9d829d2b55044c0@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[14705]:0xbb518d
q_wasm-832dc33b11cb2c78.wasm.leptos_router::components::route::RouteContext::new::{{closure}}::h4426424bd27803a9@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[9084]:0xadfae9
q_wasm-832dc33b11cb2c78.wasm.<alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call::h66fda29838f54cd0@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[17462]:0xbefbed
q_wasm-832dc33b11cb2c78.wasm.leptos_router::components::route::RouteContext::outlet::h148194381f008844@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[13999]:0xba2550
q_wasm-832dc33b11cb2c78.wasm.leptos_router::components::routes::root_route::{{closure}}::h02bcb20d8020a47e@http://127.0.0.1:9999/q-wasm-c8658c9e80ba58b6_bg.wasm:wasm-function[10788]:0xb31cc1

```

**Comment by maccesch** [comment]
Uff! I doubt that this originates from within leptos-use as something simple like allocating a String seems to fail. Thanks for reporting this. I'm going to close the issue because I have not the slightest idea what to do here except wait and see if others have the same issue.

**Comment by obioma** [comment]
Thx for replying. Agree. When searching around for similar errors there was something with leptos update referenced, and a potential source was named by @Greg Johnston. But I must admit I didn't understand.
I also looked at leptos_use code, and my impression was it happens when the message signal is touched -> leptos or dlmalloc problem. But to be on safe side I started with leptos_use - you ;-).
Now I'm not sure if I shall raise within leptos or wait as you suggested. For the next days I wait.
